Add documentation for VRef (#1923)
yewstack/yew: website/docs/concepts/html/elements.md

description: "Both HTML and SVG elements are supported"
---

## DOM nodes

> There may be many reasons why you might want to create or manage DOM nodes manually in Yew, such as

Reviewer: ```suggestion
There are many reasons why you might want to create or manage DOM nodes manually in Yew, such as
```

when integrating with JS libraries that can cause conflicts with managed components.

Using `web-sys`, you can create DOM elements and convert them into a `Node` - which can then be 
used as a `Html` value using `VRef`:

```rust
    // ...
    fn view(&self) -> Html {
        use yew::{utils::document, web_sys::{Element, Node}};

        // Create a div element from the document

>         let div: Element = document().create_element('div').unwrap();

Reviewer: This isn't js :D
```suggestion
        let div: Element = document().create_element("div").unwrap();
```
